### PR TITLE
Feacture/pc4 5

### PR DESCRIPTION
**Pull request by Daniel-JT**
here my tickets of last documets

### CODE REVIEW

**Comment by olmedo90** [comment]
Captura de pantalla (808)

Es ACORDE al figma. 👍👍

**Comment by Daniel-JT** [comment]
good night engineer, can you check my PRs, someone touched the master branch

File: src/ui/components/LastDocuments/ItemsDocuments.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            </Row> 
+        </Row>); 
+    } 
+ 
+    
+        return ( 
+            <LastDocument containerStyles={containerStyles} title='Ultimos Documentos Publicados'
+              subtitle='Group:' subtitleTwo='Support' 
+                items={[ 
+                renderStat('Análisis de la investigación sobre radio en Bolivia ...',  '321 Pag.', 'View details' ),
```
Comment (Ditmar) [inline src/ui/components/LastDocuments/ItemsDocuments.js:51]: whats this!!!

**Review by Ditmar — COMMENTED** [review]
needs work

**Comment by Daniel-JT** [comment]
image

**Review by Ditmar — COMMENTED** [review]
solve this conflic

**Review by Ditmar — APPROVED** [review]

**Pull request merged**